appveyor.yml

```diff
@@ -20,7 +20,6 @@ environment:
       TEST_MODE: fast
       APPVEYOR_SAVE_CACHE_ON_ERROR: true
       APPVEYOR_SKIP_FINALIZE_ON_EXIT: true
-      BUILD_COMMIT: v1.13.0rc2
```

Comment:
Why are you removing that?

Reply:
It's not removed; it's down below (from merge conflicts). I can reset it if you want.